DesignSafe-CI/portal
# task/DES-2936 - Hide Outputs section in form for interactive jobs

## Overview: ##
Hiding the output fields on interactive apps, to bring them in alignment with our v2 apps.

## PR Status: ##


## Related Jira tickets: ##

* [DES-2936](https://tacc-main.atlassian.net/browse/DES-2936)

## Summary of Changes: ##
Bring app definitions values into the app summary sidebar, so that some values can be conditionally hidden. Also hide the Outputs page of the wizard conditionally, but that was more straightforward.

## Testing Steps: ##
1. Open any interactive app
2. Check that the Outputs values **aren't** available from the app summary sidebar.
3. Check that the Outputs values **aren't** available in the wizard
4. Open any non-interactive app
5. Check that the Outputs values **are** available from the app summary sidebar and the wizard.

## UI Photos:

## Notes: ##
